sage-ios/SAGE/SAGE/Announcement.swift

```diff
@@ -41,8 +41,13 @@ class Announcement: NSObject {
             case AnnouncementConstants.kSchool:
                 let schoolDictionary = value as! [String: AnyObject]
                 self.school = School(propertyDictionary: schoolDictionary)
-            default:
-                self.setValue(value, forKey: propertyName)
+            case AnnouncementConstants.kId:
+                self.id = value as? Int
+            case AnnouncementConstants.kTitle:
+                self.title = value as? String
+            case AnnouncementConstants.kText:
+                self.title = value as? String
+            default: break
```

Comment:
yeah because the keys are the rails keys that come back (like last_name) plus there's some keys in there that don't correspond to user properties (stuff we don't keep track of) so it was making it crash